apps/mobile-wallet/src/components/Row.tsx

```diff
@@ -39,14 +39,26 @@ export interface RowProps {
   isLast?: boolean
   style?: StyleProp<ViewStyle>
   layout?: AnimatedProps<ViewProps>['layout']
+  isHorizontal?: boolean
```

Comment:
Wouldn't it be more logical to call this `isVertical` when we're stacking the first cell on top of the second cell?

Reply:
That's right